wiki/Legal/Copyright/ru.md

```diff
@@ -1,61 +1,59 @@
 
 1. Физическую или электронную подпись поставщика контента или пользователя сервиса.
-1. Название удалённого или заблокированного материала и адрес страницы Сайта, на которой данный материал был расположен.
-1. Утверждение о том, что у поставщика контента или пользователя сервиса есть добросовестное предположение, что материал был удалён или заблокирован по ошибке или из-за неправильной идентификации материала, который требовалось удалить или заблокировать.
-1. Имя, почтовый адрес, номер телефона и, при наличии, адрес электронной почты поставщика контента или пользователя сервиса, а также утверждение, что:
-  1. Данное лицо соглашается подчиниться судебному постановлению федерального суда по округу, в котором расположен адрес поставщика контента или пользователя сервиса, или, в случае, если адрес поставщика контента или пользователя сервиса расположен за пределами Соединённых Штатов, суда графства Лос-Анджелес в Калифорнии.
-  1. Данное лицо согласно получать судебные уведомления от лица, подавшего жалобу на нарушение авторского права, или его уполномоченного представителя.
-
-В случае, если Представителем получена встречная жалоба, osu! может отправить копию встречной жалобы первоначальному заявителю, уведомляя его о возможности восстановления или снятия блокировки с материала в течение 10 рабочих дней.
-В случае, если владелец авторского права не отправит поставщику контента или пользователю сервиса судебное уведомление, восстановление или снятие блокировки с материала будет проведено по усмотрению osu! в период от 10 до 14 и более рабочих дней с момента получения встречной жалобы.
+2. Название удалённого или заблокированного материала и адрес страницы Сайта, на которой данный материал был расположен.
+3. Утверждение о том, что у поставщика контента или пользователя сервиса есть добросовестное предположение, что материал был удалён или заблокирован по ошибке или из-за неправильной идентификации материала, который требовалось удалить или заблокировать.
+4. Имя, почтовый адрес, номер телефона и, при наличии, адрес электронной почты поставщика контента или пользователя сервиса, а также утверждение, что:
+   - Данное лицо соглашается подчиниться судебному постановлению федерального суда по округу, в котором расположен адрес поставщика контента или пользователя сервиса, или, в случае, если адрес поставщика контента или пользователя сервиса расположен за пределами Соединённых Штатов, суда графства Лос-Анджелес в Калифорнии.
+   - Данное лицо согласно получать судебные уведомления от лица, подавшего жалобу на нарушение авторского права, или его уполномоченного представителя.
```

Comment:
this isn't a list in English

Reply:
@cl8n thanks for mentioning. 
I tried to stick to the previous russian version since it had been approved before.  All of your comments above were present before my edits. Just minor adjustments for it to be/look cleaner. 
Regarding this particular issue, it has been written before in that way, so these two cannot be listed as item 5 and item 6 of a numbered list. They must be sublisted to the item 4 of a parent numbered list. That's why it has been formatted that way

Reply:
On top of that, for some reason, I couldn't reformat sublist to be numbered as well. For that reason, I've left it as unordered one

Reply:
I'm asking why there needs to be a nested list at all, in the original document item 4 is all one paragraph

Reply:
> I'm asking why there needs to be a nested list at all, in the original document item 4 is all one paragraph

Because it will become a pointless mess in russian. Making it one really long sentence will make it uncomprehendable and unreadable for literally anyone. I've just left it from original text which was present before me.